src/viam/app/data_client.py

```diff
@@ -773,8 +780,16 @@ async def configure_database_user(self, organization_id: str, password: str) ->
         """Configure a database user for the Viam organization's MongoDB Atlas Data Federation instance. It can also be used to reset the
         password of the existing database user.
 
+        ::
```

Comment:
Why is this `::` here? copy paste error?

Reply:
@npentrel To start the code sample!